README.md

```diff
@@ -14,39 +13,70 @@
 - Practice creating, storing, and modifying variables
 - Practice creating and using methods
 
-## Baseline
-You should write your program as a series of statements within a file `calculator.rb`. Anyone with this file should be able to run the command `$ ruby calculator.rb` to run the program. The program should ask the user for an operation (as a string symbol) and two numbers.
+## Objective
+
+We will build a calculator command line interface (CLI) that allows any user to interact with the command line to perform simple arithmetic. The program should ask the user to type in and enter in pieces of input. Based on the input that the user typed in, the program should output its evaluation.
+
+## Getting Started
+
+You should write your program as a series of statements within a file `calculator.rb`. Anyone with this file should be able to run the command `$ ruby calculator.rb` to run the program.
+
+## Implementation Requirements
+
+When a user runs this program, the program should ask the user for the following things:
+
+1. a math operation
+1. a number for the math operation
+1. a second number for the math operation
 
-## Primary Requirements
 The program should use the input operation and two numbers to provide the result of applying the operation to the two numbers.
+
 - The program should have support for these four operations: addition, subtraction, multiplication, and division.
-- The program should accept both the name (`add`) and the symbol (`+`) for each possible operation.
-- The program should handle unexpected user input
+- The program should accept both the name and the symbol for each possible operation. The expected names and symbols for each operation are the following:
```

Comment:
Question: Should the user input be considered case insensitive? Or is that not part of the program requirement? e.g. `add` versus `Add`.